Intersect.Client.Core/Core/Input.cs

```diff
@@ -138,6 +139,11 @@ public static void OnKeyPressed(Keys modifier, Keys key)
                 return;
             }
 
+            if (DragAndDrop.IsDragging)
+            {
+                return;
+            }
```

Comment:
What is this supposed to accomplish?

Reply:
I started doing the review, and line 142 in my opinion only blocks the open menu/close all windows, while line 205 only blocks the GUI's toogle, although the conditions are the same, the first one does not cancel the other, I tested with breakpoints, with the code as it is, maybe it is the bad arrangement of how the code is in the input, with controls being checked in separate ifs while others are checked in loops, but as it is, it is how it works, I need both if, inside the open menu and in the toggle gui